clients/roscpp/include/ros/service.h

```diff
@@ -58,40 +58,77 @@ namespace service
  * @param service_name The name of the service.
  * @param req The request message.
  * @param[out] res Storage for the response message.
+ * @param timeout The amount of time to wait for.  If timeout is -1, waits until the call is complete
```

Comment:
To be precise, it should say "If timeout is <= 0, waits until the call is complete"
Same for other occurrences...

Reply:
Done